Dockerfile

```
@@ -1,4 +1,4 @@
-FROM openjdk:11.0.15-jre as runner
+FROM openjdk:21-slim as runner
 WORKDIR runner
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)